src/css/custom.css

```diff
@@ -69,6 +76,20 @@ html[data-theme='dark'] .header-github-link:before {
 
 #mermaid-svg-0,
 #mermaid-svg-1,
-#mermaid-svg-2 {
+#mermaid-svg-2,
+#mermaid-svg-3,
+#mermaid-svg-4,
```

Comment:
This spaghetti can only be improved by adding SCSS. Unfortunately the mermaid svgs do not contain a special class we could select.

With SCSS, we can make use of the [for loop](https://sass-lang.com/documentation/at-rules/control/for).